src/math/PercentageMath.sol

```diff
@@ -10,8 +10,6 @@ library PercentageMath {
 
     uint256 internal constant PERCENTAGE_FACTOR = 1e4; // 100.00%
```

Comment:
IMO, this name is not perfect.. But I think that we want to have the same interface than Aave here right ?

Reply:
yes

Reply:
Ok